java/http/vertx/java-http-vertx-producer/pom.xml

```diff
@@ -1,14 +1,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    
     <parent>
         <groupId>io.strimzi</groupId>
         <artifactId>java-http-vertx-examples</artifactId>
         <version>1.0-SNAPSHOT</version>
     </parent>
     <artifactId>java-http-vertx-producer</artifactId>
-    
```

Comment:
I would keep the emptyline here as it separates two different sections.

Reply:
Reverted